app/src/main/res/value-it/strings.xml

```diff
@@ -0,0 +1,278 @@
+    <string name="greeting">Ciao!</string>
+    <string name="enter_credentials">Esegui il login con le credenzali Warmshowers.org</string>
+    <string name="view_details">Vedi dettagli</string>
+    <string name="star">Stella</string>
+    <string name="close">Vicino</string>
+    <string name="username_hint">Username o Email</string>
+    <string name="password_hint">Password</string>
+    <string name="button_text_login">Login</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Cancella</string>
+    <string name="description">Info sul profilo</string>
+    <string name="last_login"><u>Ultimo login:</u></string>
+    <string name="member_since"><u>Membro dal:</u></string>
+    <string name="location"><u>Posizione:</u></string>
+    <string name="mobile_phone"><u>Numero Cellulare:</u></string>
+    <string name="preferred_notice"><u>Notizie preferite:</u></string>
+    <string name="max_guests"><u>Massimo di ospiti:</u></string>
+    <string name="nearest_accomodation"><u>Alloggio:</u></string>
+    <string name="campground"><u>Campeggio:</u></string>
+    <string name="bike_shop"><u>Negozio di bici:</u></string>
+    <string name="services"><u>Può fornire:</u></string>
+    <string name="home_phone"><u>Numero di casa:</u></string>
+    <string name="work_phone"><u>Numero del lavoro:</u></string>
+    <string name="update">Aggiorna</string>
+    <string name="delete">Elimina</string>
+    <string name="account">Account</string>
+    <string name="settings">Opzioni</string>
+    <string name="about">Informazioni</string>
+    <string name="last_updated">Ultimo aggiornamento:</string>
+    <string name="host_updated">Informazioni sull'ospite aggiornate.</string>
+    <string name="host_not_available">Questo membro non è momentaneamente disponibile. Puoi comunque contattarlo, ma non per l'ospitalità.</string>
+	<!-- QUI -->
+    <string name="unstar_host">Un-favorite this user?</string>
+    <string name="user_starred">Member favorited. You can now access this information offline.</string>
+    <string name="user_unstarred">Member un-favorited.</string>
+    <string name="performing_search">Performing search …</string>
+    <string name="io_error">Internet access failure, please check your Internet connection.</string>
+    <string name="http_unexpected_failure">Unexpected error accessing server. Check your credentials and Internet connection.</string>
+    <string name="subject_hint">Enter subject here</string>
+    <string name="message_hint">Enter message here</string>
+    <string name="message_validation_error">Both subject and message are required.</string>
+    <string name="sending_message">Sending message …</string>
+    <string name="error_sending_message">Could not send message. Check your credentials and internet connection.</string>
+    <string name="message_sent">Message sent successfully.</string>
+    <string name="about_1">Developed by Warmshowers.org volunteers.</string>
```

Comment:
As a quick heads-up: Here, we mention the translators at the end. So in non-English languages this string is "Developed by Warmshowers.org volunteers. \nTranslation by: [Your Name] (Italian)".